apps/golden-sample-app-e2e/config/ebp-sndbx.config.json

```diff
@@ -0,0 +1,12 @@
+{
+    "headerKey": "x-sdbxaz-api-key",
+    "headerValue": "apisandbox-d0d8278f-5fc5-46fb-5fc5-d0b6e1cc8059",
+    "users": {
+      "userWithNoContext": {
```

Comment:
As agreed, for the golden sample these can go to secrets, while that is not required for rnd-internal repositories